scripts/funcotator/data_sources/getGencodeXRefseq.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+outFileBaseName="gencode_xrefseq"
+outExt=".tsv"
+
+hg19db="homo_sapiens_core_75_37"
+hg38db="homo_sapiens_core_90_38"
+
+hg19FileName=${outFileBaseName}_v75_37.hg19${outExt}
+hg38FileName=${outFileBaseName}_v90_38.hg38${outExt}
+
+################################################################################
+
+
+# Create our query to the DB:
```

Comment:
If we don't need what is in the comment, please delete.  If we do, please explain what it is.

Reply:
Fixed!